src/@components/JoinPage/UserProfilePage/ProfileBirth/index.tsx

```diff
@@ -0,0 +1,82 @@
+import { useState } from "react";
+import DatePicker from "react-mobile-datepicker";
+
+import { IcDownArrow } from "../../../../asset/icon";
+import { St } from "./style";
+
+interface birthTypes {
+  isbirth: string;
+  setIsbirth: (birthDay: string) => void;
+}
+
+export default function ProfileBirth(props: birthTypes) {

```

Comment:
⚡반영 필요⚡--> `commit suggestion`

props 타입 컨벤션 지켜주세요!

```suggestion
interface ProfileBirthProps {
  isbirth: string;
  setIsbirth: (birthDay: string) => void;
}

export default function ProfileBirth(props: ProfileBirthProps) {
```

Reply:
넵!!!!!😎😎😎